builder/index.html

```diff
@@ -65,11 +65,8 @@ <h3 class="modal-title">Clone Technique</h3>
               <div class="form-group">
                 <label for="cloneName" class="control-label">New Technique name:</label>
                 <div>
-                  <input  type="text" id="cloneName" name="name" techniquename class="form-control" ng-model="technique.name" placeholder="Technique Name" required ng-maxlength="2048">
+                  <input  type="text" id="cloneName" name="name" techniquename class="form-control" ng-model="technique.name" placeholder="Technique Name" required>
                 </div>
-                <span class="text-danger" ng-show="cloneForm.name.$error.maxlength">
-                  Fields over 2048 characters are currently not supported. Please insert your content into a file, and copy it with a file_copy_* method, or use a template.
```

Comment:
Because the Technique name is only used in the application so it does not require length check.
On the other hand, this check has to be done on the Technique bundle name which is used in the URL.